src/ui/pages/Menu/components/Settings/Settings.tsx

```diff
@@ -170,35 +176,39 @@ const Settings = () => {
   };
 
   const handleOptionClick = async (item: OptionProps) => {
-    setOption(item.label);
-    switch (item.label) {
-    case i18n.t("settings.sections.security.biometry"): {
+    setOption(item.index);
+    switch (item.index) {
+    case 0: {
       handleBiometricUpdate();
       break;
     }
-    case i18n.t("settings.sections.security.changepin.title"): {
+    case 1: {
       handleChangePin();
       break;
     }
+    case 2: {
```

Comment:
imo this has poor readability. It would be better if there was an enum instead of an index.